indico/modules/events/contributions/controllers/management.py

```diff
@@ -679,3 +680,10 @@ def _process_POST(self):
         self.subcontrib.references.append(reference)
         db.session.flush()
         return self.jsonify_reference(reference)
+
+
+class RHCloneContribution(RHManageContributionBase):
+    def _process(self):
+        clone_contribution(self.event, self.contrib, False)
+        db.session.commit()
```

Comment:
why are you committing here? `db.session.flush()` is all you need if you just need to populate the id and default args